package.json

```diff
@@ -19,21 +20,24 @@
   "license": "ISC",
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "build-lib": "mkdir -p ./dist && rm -rf ./dist/* && cp ./src/freestarAdSlot.vue ./dist/index.vue  && cp ./src/freestarWrapper.js ./dist/freestarWrapper.js",
-    "publish": "npm publish --access public"
+    "build": "vue-cli-service build --target lib --name pubfig-adslot-vue src/index.js && rm ./dist/demo.html"
   },
   "dependencies": {
-    "core-js": "^2.6.5",
-    "vue": "^2.6.10"
+    "lodash.isequal": "^4.5.0",
+    "lodash.sortby": "^4.7.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.13.15",
+    "@babel/preset-env": "^7.13.15",
     "@vue/cli-plugin-babel": "^3.9.0",
     "@vue/cli-plugin-eslint": "^3.9.0",
     "@vue/cli-service": "^3.9.0",
+    "@vue/eslint-config-airbnb": "^5.3.0",
     "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.2.2",
     "eslint": "^5.16.0",
     "eslint-plugin-vue": "^5.0.0",
+    "vue": "^2.6.12",
```

Comment:
things that are needed to run but not for the build should be listed as peer dependencies. This will ensure the plugin users doesnt use an incompatible version of vue for example. See the react plugin for reference